lib/filecoin-rpc-client.js

```diff
@@ -35,6 +36,7 @@ export async function getIndexProviderPeerIdFromFilecoinMinerInfo(
       // The maximum amount of attempts until failure.
       retries: maxAttempts,
     })
+    assert.ok(typeof res.PeerId === 'string', 'PeerId must be a string')
```

Comment:
```suggestion
    assert.ok(typeof res.PeerId === 'string', `PeerId must be a string, is of type "${typeof PeerId}"`)
```

Reply:
Should we use `assert.strictEqual` instead?